test/integration/buildcontroller_test.go

```diff
@@ -165,7 +165,10 @@ func setupBuildControllerTest(counts controllerCount, t *testing.T) (*client.Cli
 		}
 	}
 	for i := 0; i < counts.ImageChangeControllers; i++ {
-		openshiftConfig.RunImageTriggerController()
+		_, err := openshiftControllerInitializers["imagetrigger"](openshiftControllerContext)
```

Comment:
I'm guessing this should be `image-trigger` since in the 3rd commit you're changing that.